### PR TITLE
Update CIP-59.md

### PR DESCRIPTION
**Pull request by oed**

### CODE REVIEW

File: CIPs/CIP-59/CIP-59.md [hunk 1]
```diff
@@ -12,58 +12,81 @@ edited: 2020-09-24
...
+<streamid> ::= <multibase-prefix><multicodec-streamid><stream-type><multicodec-cidv1><multicodec-content-type><multihash-content-address>
 ```
 
 Where
 
-- `<multibase-prefix>` is a [multibase](https://github.com/multiformats/multibase) code (1 or 2 bytes), to ease encoding DocIDs into various bases. **NOTE:** *Binary* (not text-based) protocols and formats may omit the multibase prefix when the encoding is unambiguous.
-- `<multicodec-docid>` is a [multicodec](https://github.com/multiformats/multicodec) used to indicate that it's a DocID.
-- `<doctype>` is a [varint](https://github.com/multiformats/unsigned-varint) representing the doctype of the document.
+- `<multibase-prefix>` is a [multibase](https://github.com/multiformats/multibase) code (1 or 2 bytes), to ease encoding StreamIDs into various bases. **NOTE:** *Binary* (not text-based) protocols and formats may omit the multibase prefix when the encoding is unambiguous.
+- `<multicodec-streamid>` is a [multicodec](https://github.com/multiformats/multicodec) used to indicate that it's a StreamID.
```
Comment (stbrody) [inline CIPs/CIP-59/CIP-59.md:41]: I guess we'll need to make a PR to update https://github.com/multiformats/multicodec/blob/master/table.csv as well

File: CIPs/CIP-7/CIP-7.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The **genesis record** is stored in IPLD using the *dag-cbor* codec. It has the
 
 No other properties are allowed to be defined.
 
-**Signed records** of a CAIP-10 Link document are also encoded using *dag-cbor*. Other than that they conform to the standard record format. The content of the `data` property should be a *proof* as returned by the [3id-blockchain-utils](https://github.com/3box/js-3id-blockchain-utils) library. This library has a general structure for creating and verifying "link proofs" for any blockchain. 
+**Signed records** of a CAIP-10 Link stream are also encoded using *dag-cbor*. Other than that they conform to the standard record format. The content of the `data` property should be a *proof* as returned by the [3id-blockchain-utils](https://github.com/3box/js-3id-blockchain-utils) library. This library has a general structure for creating and verifying "link proofs" for any blockchain.
```
Comment (stbrody) [inline CIPs/CIP-7/CIP-7.md:46]: Should we update `record` to `commit` throughout this page while we're here?

File: CIPs/CIP-8/CIP-8.md [hunk 3]
```diff
@@ -12,24 +12,24 @@ requires: Doctypes (CIP-5)
 
 ## Simple Summary
 
-This CIP describes the Tile doctype which is a very general document type that has JSON as it's content and is controlled by a [DID](https://w3c.github.io/did-core/).
+This CIP describes the Tile document which is a very general stream type that has JSON as it's content and is controlled by a [DID](https://w3c.github.io/did-core/).
```
Comment (stbrody) [inline CIPs/CIP-8/CIP-8.md:15]:
```suggestion
This CIP describes the Tile document which is a very general stream type that has JSON as its content and is controlled by a [DID](https://w3c.github.io/did-core/).
```

File: CIPs/CIP-8/CIP-8.md [hunk 4]
```diff
@@ -12,24 +12,24 @@ requires: Doctypes (CIP-5)
...
 
 
 ## Abstract
 
-The Tile doctype allows DIDs to create simple JSON documents that use [json-patch](http://jsonpatch.com/) for updating the content of the document. Each Tile is owned by one or multiple DIDs, all of which need to sign the update record. The signed records are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) which allows the signatures to be natively encoded directly in [IPLD](ipld.io). Tiles also support adding schemas to the document which means that the json data within the content of the document can be enforced to have a specific format.
+The Tile document allows DIDs to create simple JSON documents that use [json-patch](http://jsonpatch.com/) for updating the content of the document. Each Tile is owned by one or multiple DIDs, all of which need to sign the update record. The signed records are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) which allows the signatures to be natively encoded directly in [IPLD](ipld.io). Tiles also support adding schemas to the document which means that the json data within the content of the document can be enforced to have a specific format.
```
Comment (stbrody) [inline CIPs/CIP-8/CIP-8.md:20]: I wonder if we should remove the language about multiple DIDs for now?  Until we actually support multiple controllers and know what the semantics will be

**Review by stbrody — COMMENTED** [review]
looks good, just a couple of questions about other possible updates to these CIPs that also need making

File: CIPs/CIP-7/CIP-7.md (hunk 2 above)
Comment (oed) [reply CIPs/CIP-7/CIP-7.md:46]: yes, thanks.

File: CIPs/CIP-8/CIP-8.md (hunk 4 above)
Comment (oed) [reply CIPs/CIP-8/CIP-8.md:20]: I didn't really look so deeply here. Could definitely do that.

File: CIPs/CIP-8/CIP-8.md [hunk 5]
```diff
@@ -12,113 +12,112 @@ requires: Doctypes (CIP-5)
...
 
 
 ## Abstract
 
-The Tile doctype allows DIDs to create simple JSON documents that use [json-patch](http://jsonpatch.com/) for updating the content of the document. Each Tile is owned by one or multiple DIDs, all of which need to sign the update record. The signed records are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) which allows the signatures to be natively encoded directly in [IPLD](ipld.io). Tiles also support adding schemas to the document which means that the json data within the content of the document can be enforced to have a specific format.
+The Tile document allows DIDs to create simple JSON documents that use [json-patch](http://jsonpatch.com/) for updating the content of the document. Each Tile is owned by a DID, which need to sign the update commit. The signed commits are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) which allows the signatures to be natively encoded directly in [IPLD](ipld.io). Tiles also support adding schemas to the document which means that the json data within the content of the document can be enforced to have a specific format.
```
Comment (stbrody) [inline CIPs/CIP-8/CIP-8.md:20]:
```suggestion
The Tile document allows DIDs to create simple JSON documents that use [json-patch](http://jsonpatch.com/) for updating the content of the document. Each Tile is owned by a DID, which needs to sign the update commit. The signed commits are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) which allows the signatures to be natively encoded directly in [IPLD](ipld.io). Tiles also support adding schemas to the document which means that the json data within the content of the document can be enforced to have a specific format.
```

File: CIPs/CIP-8/CIP-8.md [hunk 6]
```diff
@@ -12,113 +12,112 @@ requires: Doctypes (CIP-5)
...
 ```js
 state.next = {
-  content: applyJSONPatch(state.content, signedRecord.data),
+  content: applyJSONPatch(state.content, signedCommit.data),
   metadata: {
-    schema: signedRecord.schema,
-    family: genesisRecord.family,
-    tags: signedRecord.tags
+    schema: signedCommit.schema,
+    family: genesisCommit.family,
```
Comment (stbrody) [inline CIPs/CIP-8/CIP-8.md:93]: is this trying to imply that `family` can't be changed?  Because we do actually allow changing `family` in the new API

**Review by stbrody — APPROVED** [review]

File: CIPs/CIP-8/CIP-8.md (hunk 6 above)
Comment (oed) [reply CIPs/CIP-8/CIP-8.md:93]: I guess. Don't want to get into a discussion about what the correct behavior should be. Can we keep focus on the main things this PR is changing, and do other changes separately / later.

**Pull request merged**